package.json

```diff
@@ -55,35 +57,34 @@
 		"meow": "^12.0.1",
 		"node-fetch": "^3.3.1",
 		"pascal-case": "^3.1.2",
-		"prettier": "^2.8.8",
-		"ts-morph": "^18.0.0"
+		"quick-lru": "^6.1.1"
 	},
 	"devDependencies": {
-		"@prismicio/client": "^7.0.0",
-		"@prismicio/mock": "^0.2.0",
-		"@prismicio/types": "^0.2.8",
-		"@size-limit/preset-small-lib": "^8.2.4",
+		"@prismicio/client": "^7.1.0",
+		"@prismicio/mock": "^0.3.1",
+		"@size-limit/preset-small-lib": "^8.2.6",
 		"@trivago/prettier-plugin-sort-imports": "^4.1.1",
 		"@types/common-tags": "^1.8.1",
-		"@types/prettier": "^2.7.2",
-		"@typescript-eslint/eslint-plugin": "^5.59.6",
-		"@typescript-eslint/parser": "^5.59.6",
-		"@vitest/coverage-c8": "^0.31.1",
-		"eslint": "^8.40.0",
+		"@types/find-cache-dir": "^3.2.1",
+		"@typescript-eslint/eslint-plugin": "^5.60.1",
+		"@typescript-eslint/parser": "^5.60.1",
+		"@vitest/coverage-c8": "^0.33.0",
+		"eslint": "^8.43.0",
 		"eslint-config-prettier": "^8.8.0",
 		"eslint-plugin-prettier": "^4.2.1",
 		"eslint-plugin-tsdoc": "^0.2.17",
+		"prettier": "^2.8.8",
 		"prettier-plugin-jsdoc": "^0.4.2",
-		"size-limit": "^8.2.4",
+		"prismic-ts-codegen-v0-1-11": "npm:prismic-ts-codegen@0.1.11",
+		"size-limit": "^8.2.6",
 		"standard-version": "^9.5.0",
-		"typescript": "^5.0.4",
-		"vite": "^4.3.7",
+		"typescript": "^5.1.3",
+		"vite": "^4.3.9",
 		"vite-plugin-sdk": "^0.1.1",
-		"vitest": "^0.31.1"
+		"vitest": "^0.33.0"
 	},
 	"peerDependencies": {
-		"@prismicio/client": "^6.6 || ^7",
-		"@prismicio/types": "^0.2.8"
```

Comment:
Maybe we remove it from `peerDependenciesMeta` also then? Not sure why we removed it though 🤔 Do we drop support already?

Reply:
It was probably removed accidentally. `@prismicio/types` should remain since we still support it. It's been restored. 🙂